src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const oneDayCost = 40;
+  const shortTerm = 3;
+  const longTerm = 7;
+  const shortTermDiscount = 20;
+  const longTermDiscount = 50;
+
+  if (days < shortTerm) {
+    return days * oneDayCost;
+  }
+
+  if (days < longTerm) {
+    return days * oneDayCost - shortTermDiscount;
+  }
+
+  return days * oneDayCost - longTermDiscount;
```

Comment:
Consider adding a condition `else if (days >= longTerm)` before this return statement for clarity, even though it's logically correct as it is. This will make the code more readable and explicit about the conditions being checked.